third_party/java/java_tools/update_java_tools.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# A script to update the checked-in jars corresponding to Java tools used
+# by the Java rules in Bazel.
+#
+# For usage please run
+# ~/third_party/java/java_tools/update_java_tools.sh help
```

Comment:
nit: omit `~/` (`~` means your home directory)

Reply:
Done. I added it out of habit :confused:

Reply:
(I'm still seeing `~`)